remote/src/main/java/org/buffer/android/boilerplate/remote/model/ArticleModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.buffer.android.boilerplate.remote.model
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Representation for a [ArticleModel] fetched from the API
+ */
+class ArticleModel(
```

Comment:
For remote (or data) module. The naming would be like this:
1. domain -> Article
2. data/remote/cache -> ArticleEntity (db), ArticleResponse/ArticleRequest/ArticleParam (network/json)
3. ui/presentation -> ArticleIntent

Why I choose using `ArticleIntent` instead of `ArticleView` because we might creating a custom view, thus will be conflicted with POJO/data class.

cc @adesamp